src/app/routes/getInitialData/article.js

```diff
@@ -3,21 +3,18 @@ import getBaseUrl from './utils/getBaseUrl';
 import fetchData from './utils/fetchData';
 
 const getArticleInitialData = async ({ match }) => {
-  const { id, service, amp } = match.params;
-  const isAmp = !!amp;
+  const { id, service } = match.params;
 
   const baseUrl = onClient()
     ? getBaseUrl(window.location.origin)
     : process.env.SIMORGH_BASE_URL;
 
   const url = `${baseUrl}/${service}/articles/${id}.json`;
 
-  const { data, status } = await fetchData({ url });
+  const { pageData, status } = await fetchData({ url });
```

Comment:
Were expanding this object and then rebuilding it below, we can just return directly :)

Reply:
Good spot.